### PR TITLE
Update documentation within the user guide for the spectrum viewer

### LINKED ISSUE #2048: Spectrum Viewer Documentation
### Desired Behaviour

Improve document for the Spectrum Viewer to reflect all functionality available within the Spectrum Viewer

### Current Behaviour

Only basic documentation for the Spectrum Viewer currently exists

### PR DESCRIPTION
**Pull request by JackEAllen**
### Issue

Closes #2048 

### Description

Update Spectrum Viewer Documentation

### Testing 

* Proof Read to check for spelling errors
* Build Docs to check updated documentation is correctly formatted.
* Check URLs are correct.

### Acceptance Criteria 

- [ ] Documentation describes the spectrum viewer today.
- [ ] Documentation provides a good description of the spectrum viewer to someone who has never used the spectrum viewer who may have no prior experience working with tomography data. 
- [ ] No spelling or grammar errors
- [ ] All URLs work

### Documentation

`docs/release_notes/next/feature-2048-spectrum_documentation`

### CODE REVIEW

File: docs/user_guide/gui/spectrum_viewer.rst [hunk 1]
```diff
@@ -1,29 +1,98 @@
...
 
+The Spectrum Viewer has two main modes of operation: "ROI" mode and "Image" mode.
+
+
+ROI Mode
+----------
+ROI mode is the default mode of the Spectrum Viewer. In this mode, you can add, rename and remove many ROIs to and from the spectrum viewer and view the spectrum of each ROI.
+The spectrum viewer can be used to export the spectrum of each ROI and its respective coordinates to a separate csv file.
+
+To add an ROI, click the "Add" button and then click and drag the mouse over the newly created ROI within sample image to resize and position the ROI.
```
Comment (MikeSullivan7) [inline docs/user_guide/gui/spectrum_viewer.rst:33]:
```suggestion
To add an ROI, click the "Add" button and then click and drag the mouse over the newly created ROI within the sample image to resize and position the ROI.
```

File: docs/user_guide/gui/spectrum_viewer.rst [hunk 2]
```diff
@@ -1,29 +1,98 @@
...
+The export spectrum button will open a file dialog where you can select the directory and file name to save the csv file and is only enabled if you have selected a valid tomography data stack and open beam data stack.
+Both stacks must be the same shape and size. If the open beam is a different shape or size, the export spectrum button will be disabled and a warning message will be displayed.
+
+
+Image Mode
+----------
+Image mode, accessible by selecting the "Image" tab, is a mode of the Spectrum Viewer where you can view the spectrum within a singular region of interest (ROI) and export the spectrum in a format compatible with `RITS <https://mlfinfo.jp/bl22/en/GUI-RITS.html>`_ .
+In this mode, you can select whether you would like to export your data in single spectrum or binned spectrum format. 
+You can also select the step size and bin size. If the step size if the same as the bin size, a tiled average will be performed. if the step size is less than the bin size, a rolling average will be performed. The unit of measurement is pixels
```
Comment (MikeSullivan7) [inline docs/user_guide/gui/spectrum_viewer.rst:46]:
```suggestion
You can also select the step size and bin size. If the step size if the same as the bin size, a tiled average will be performed. If the step size is less than the bin size, a rolling average will be performed. The unit of measurement is pixels.
```

File: docs/user_guide/quick_start.rst [hunk 3]
```diff
@@ -125,11 +125,34 @@ you can apply some post-processing operations (such as circular mask), or just s
...
+
+#. If you have a TOF dataset loaded where you would like to normalise to the open beam, you can do so by toggling the "Normalise to open beam" checkbox. This will normalise the spectrum to the open beam spectrum. The greyed out drop down menu will become available and you can select which dataset you would like to use as the open beam.
+
+**Export Single or Binned Spectrum Data in RITS Format**
+
+#. To enter IMAGE mode to export single or binned spectrum data in RITS format, you can click the "IMAGE" tab at the top of the table of ROIs.
+
+#. Resize the ROI in the same way as before for exporting regular spectrum data to cover an area of the sample you would like to acquire the spectrum for.
+
+#. Select the output mode from the top drop down menu on the left which replaces the table of ROIs, either "Single Spectrum" or 2D Binned" depending on what you would like to export. 2D binned will bin within the roi and export the binned data as multi-spectrum data.
```
Comment (MikeSullivan7) [inline docs/user_guide/quick_start.rst:154]:
```suggestion
#. Select the output mode from the top drop down menu on the left which replaces the table of ROIs, either "Single Spectrum" or 2D Binned" depending on what you would like to export. 2D binned will bin within the ROI and export the binned data as multi-spectrum data.
```

**Review by MikeSullivan7 — CHANGES_REQUESTED** [review]
Looks great, just a few small typos :)

Would it be useful at all to include the error propagation equation which is being used in the Error Propagation mode?

**Comment by JackEAllen** [comment]
> Looks great, just a few small typos :)
> 
> Would it be useful at all to include the error propagation equation which is being used in the Error Propagation mode?

Because propagated Error is pretty standard, I think we should leave it out of our docs for the time being. If we get a user request in the future asking how we calculate it, then we can probably add it to our docs. I thinks the written description does a pretty good job of describing what error propagation is.

**Review by MikeSullivan7 — APPROVED** [review]

**Pull request merged**